frontend/src/components/PackageGeneration/Editor.js

```diff
@@ -45,7 +45,11 @@ class editor extends React.Component {
     xhr.open("POST", 'http://localhost:8080/package/downloadWarPackage', true);
     xhr.responseType = "blob";
     xhr.onload = function () {
+      if(xhr.status == 404) {
+        alert("This is something wrong with your configuration file. Please fix and try again")
```

Comment:
If there's a 404, you should send that error up to the main program

Reply:
Yeah this is the highest we can go right? The next would be displaying an alert on the UI.